### DISCUSSION TITLE
Add LG .d.ts files to github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types

### DISCUSSION THREAD

**Discussion by CCCullens** [sachinchoolur/lightGallery#1127] [General]
This allows typesafe use without installing all of LG. Developers can use npm install @types/lightgallery. That is, only adding a few files to node_modules. 
The necessary css and js files can be accessed at runtime from cdn.jsdeliver.net

**Comment by sachinchoolur** [comment]
Importing type declaration files from node_modules doesn't increase bundle size. 

DefinitelyTyped generally contains type declaration for libraries written in pure JavaScript. Since type declaration files are already available in lightGallery repo, we don't have to publish it again to DefinitelyTyped  

Let me know if you have any question
